### PR TITLE
Directive movie details bf-024 ready for testing

### PR DESCRIPTION
**Pull request by axelNolasco**
The file movie-details.js has some code commented for future use

### CODE REVIEW

File: blimflix/client/app/directives/movie-details/movie-details.scss [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+.info h3 {
+    margin: 0;
+    display: inline-block;
+}
+
+.rank {
+
+}
+
+.staff h6{
```
Comment (devRodSoft) [inline blimflix/client/app/directives/movie-details/movie-details.scss]: Plase add space after h6

File: blimflix/client/app/directives/movie-details/movie-details.scss [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    margin-bottom: 0px;
+    color: #999;
+    font-weight: bold;
+}
+
+.staff p{
```
Comment (devRodSoft) [inline blimflix/client/app/directives/movie-details/movie-details.scss]: Add space after p

File: blimflix/client/app/directives/movie-details/movie-details.scss [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+.info div {
+    padding: 0;
+}
+
+.info h3 {
+    margin: 0;
+    display: inline-block;
+}
+
+.rank {
```
Comment (CarlosPadilla) [inline blimflix/client/app/directives/movie-details/movie-details.scss]: this class has no contents. if it isn't required then remove it

**Review by CarlosPadilla — APPROVED** [review]

**Comment by lordpixel** [comment]
Hi @Axl-Nolasco, how can we display the directive?

**Comment by axelNolasco** [comment]
Hi Manu. In the file .js there are some lines of code of a button with specific classes that display the modal. If you put this code in the all movies html you can visualize it.



On Sun, Dec 4, 2016 at 4:16 PM -0600, "Manuel Ramirez" <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi @Axl-Nolasco<https://github.com/Axl-Nolasco>, how can we display the directive?

-

**Review by devRodSoft — APPROVED** [review]

File: blimflix/client/app/directives/movie-details/movie-details.scss [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+    background-color: white;
+    margin-left: 15px;
+    height: 280px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    margin-right: 15px;
+}
+
+.info div {
```
Comment (CarlosPadilla) [inline blimflix/client/app/directives/movie-details/movie-details.scss]: please nest the repeated  data

**Review by CarlosPadilla — CHANGES_REQUESTED** [review]

File: blimflix/client/app/directives/movie-details/movie-details.scss (hunk 4 above)
Comment (CarlosPadilla) [reply blimflix/client/app/directives/movie-details/movie-details.scss]: this changes can affect all over the application. please make it nested
now you are setting global style changes for elements

**Review by CarlosPadilla — CHANGES_REQUESTED** [review]

**Review by CarlosPadilla — APPROVED** [review]

**Pull request merged**